portfolio/src/main/webapp/index.html

```diff
@@ -5,14 +5,58 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <style> 
+      body{ background-color: black; }
+      h1{ color: white;}
+      p{ color: white}
+      h2{ color: white;}
+      ul{color: white;}
+      ol{color: white;}
+      a{color: white;}
+    </style>    
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>SPS Portfolio</h1>
       <p>This is my portfolio.</p>
       <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <p>Spring 2020</p>
+
+    <p>Hello World! I'm Cameron Payton. 
+        I'm a current Sophomore computer science major at Hampton University. 
+        As you already know, I like to code. 
+        But other things I like to do in my free time are watch anime, play video games, workout, and study philosophy.</p>
+    <img align="bottom" src ="https://orig00.deviantart.net/f480/f/2013/246/d/d/goodnight_punpun_by_double_doubleyou-d6kwt9z.png" 
+      width = "150" height = "150" alt="Pic of Onodera PunPun"/>
+    <img align="bottom" src ="https://www.bing.com/th?id=OIP.h60tulDWmY-R1y1EPXjSrQHaKg&w=143&h=204&c=8&rs=1&qlt=90&dpr=1.5&pid=3.1&rm=2"
+      width = "150" height = "150" alt="Pic of atom!"/>
+    <img align="bottom" src="https://th.bing.com/th/id/OIP.ARuZ6HfGiuN3M53EGMwSjQHaKf?w=200&h=284&c=7&o=5&dpr=1.5&pid=1.7"
+    width = "150" height = "150" alt="Pics of my favorite megaman game"/>
+    <img align="bottom" src="https://th.bing.com/th/id/OIP.6m1Bj4_fuuq6JUwibVyZ_AHaFj?w=275&h=206&c=7&o=5&dpr=1.5&pid=1.7"  
+    width = "150" height = "150" alt="Pic of Shinji Ikari"/>
+        <h2> Favorite Anime/Manga</h2>
+        <ul> 
+            <li>Neon Genesis Evangelion</li>
+            <li>Goodnight Pun-Pun</li>
+            <li>Pluto</li>
+            <li>Megaman</li>
+            <li>The Horizon</li>
+        </ul>
+        
+    <h2> Current Workouts</h2>
+        <ul>
+            <li>Barbell Squat</li>
```

Comment:
Ooh is this Stronglifts 5x5? (Or some variant thereof?) This is identical to the routine I had when I was starting to work out! (When I worked out, at least...)